etc/cvmfs/domain.d/eessi.io.conf

```diff
@@ -0,0 +1,8 @@
+# Stratum 1 servers for the eessi.io domain
+CVMFS_SERVER_URL="http://aws-eu-central-s1.eessi.science/cvmfs/@fqrn@;http://azure-us-east-s1.eessi.science/cvmfs/@fqrn@"
+
+# Public keys for the eessi.io domain
+CVMFS_KEYS_DIR="$CVMFS_MOUNT_DIR/$CVMFS_CONFIG_REPOSITORY/etc/cvmfs/keys/eessi.io"
```

Comment:
@boegel As mentioned on Slack, I think I do agree with @DrDaveD's suggestion: since we're using a Yubikey now, we will be using the same key for all `*.eessi.io` repos, so it makes sense to reflect that in the filename of the public key. We should probably even move it a directory up, as there's no need for having a separate directory anymore for just a single key? So:


```suggestion
# Public key for the eessi.io domain
CVMFS_PUBLIC_KEY="$CVMFS_MOUNT_DIR/$CVMFS_CONFIG_REPOSITORY/etc/cvmfs/keys/eessi.io.pub"
```

Reply:
I would have no objection to moving it up a level, but all the other domains in the default config repo have a directory so I think Jakob at least would prefer it in a directory.  Let's stick with existing convention.

Reply:
Indeed, see also `etc/cvmfs/keys/opensciencegrid.org/opensciencegrid.org.pub`, so I'll stick to `CVMFS_KEYS_DIR` and only rename `etc/cvmfs/keys/eessi.io/software.eessi.io.pub` to `etc/cvmfs/keys/eessi.io/eessi.io.pub`

Reply:
Oh, I actually saw an example that did use the `keys` directory (https://github.com/cvmfs-contrib/config-repo/blob/master/etc/cvmfs/domain.d/skatelescope.eu.conf, https://github.com/cvmfs-contrib/config-repo/tree/master/etc/cvmfs/keys), but leaving it in a subdirectory is okay with me as well :-)